server/testing/src/main/java/org/apache/james/utils/IMAPMessageReader.java

```diff
@@ -82,6 +77,27 @@ public IMAPMessageReader awaitMessage(ConditionFactory conditionFactory) throws
         return this;
     }
 
+    public IMAPMessageReader awaitMessageCount(ConditionFactory conditionFactory, int messageCount) {
+        conditionFactory.until(() -> {
+            try {
+                imapClient.fetch("1:*", "ALL");
+                return  countFetchedEntries() == messageCount;
```

Comment:
2 spaces

Reply:
Ha...

Reply:
Funny that it's not caught by checkstyle...

Reply:
I tried to find why, but failed. But it seems that checkstyle is ignoring some tests files.